coordinate.go

```diff
@@ -223,8 +224,8 @@ func (a *Agent) updateFailedNodeTxn(node *api.Node, kvClient *api.KV, key string
 
 		// Check if the node has been critical for too long and needs to be reaped.
 		if time.Since(criticalStart) > a.config.NodeReconnectTimeout {
-			a.logger.Printf("[INFO] reaping node %q that has been failed for more then %s",
-				node.Node, a.config.NodeReconnectTimeout.String())
+			a.logger.Info("reaping node", "info", hclog.Fmt(" %q has been failed for more then %s",
+				node.Node, a.config.NodeReconnectTimeout.String()))
```

Comment:
This could be reorganized to something like
```
a.logger.Info("reaping node that has failed for too long", "node", hclog.Fmt("%q", node.Node), "failureTimeout", a.config.NodeReconnectTimeout.String())
```

Reply:
Not sure if you missed the above comment or you disagree. Please let me know!